Client/Application/AppDelegate+Experiments.swift

```diff
@@ -10,17 +10,16 @@ import Foundation
 
 extension AppDelegate {
     func initializeExperiments() {
-
         let defaults = UserDefaults.standard
         let nimbusFirstRun = "NimbusFirstRun"
-        let firstRun = defaults.bool(forKey: nimbusFirstRun) != false
+        let isFirstRun = defaults.bool(forKey: nimbusFirstRun) != false
         defaults.set(false, forKey: nimbusFirstRun)
-
+        Experiments.customTargetingAttributes =  ["isFirstRun":  "\(isFirstRun)"]
```

Comment:
Note the name `isFirstRun` to align with android (so on experimenter we have only one name and less chance for error)